build/gen_stub.php

```diff
@@ -1351,7 +1351,9 @@ public function getDeclaration(): string {
                 if (count($arginfoType->classTypes) >= 2) {
                     foreach ($arginfoType->classTypes as $classType) {
                         $className = $classType->name;
-                        $code .= "\tzend_string *property_{$propertyName}_class_{$className} = zend_string_init(\"$className\", sizeof(\"$className\") - 1, 1);\n";
+                        $escapedClassName = str_replace('\\', '\\\\', $className);
```

Comment:
I believe this is `$classType->toEscapedName()`.

Reply:
Thanks.
It seems to me that there is no `ClassType` type and everything is `SimpleType`, is this intended?
If it is intended, keep it is.

Reply:
I noticed this as well, and fixed the type hint in https://github.com/php/php-src/commit/c1c1822e8c324ccfbb2f430cf2bf20156e287ccd.